src/test/codewhisperer/service/keyStrokeHandler.test.ts

```diff
@@ -153,7 +154,11 @@ describe('keyStrokeHandler', function () {
             )
             CodeWhispererUserGroupSettings.instance.userGroup = userGroup
             await KeyStrokeHandler.instance.processKeyStroke(mockEvent, mockEditor, mockClient, config)
-            assert.strictEqual(invokeSpy.called, shouldTrigger)
+            assert.strictEqual(
+                invokeSpy.called,
+                shouldTrigger,
+                `invokeAutomatedTrigger ${shouldTrigger ? 'NOT' : 'WAS'} called for rightContext: ${rightContext}`
```

Comment:
Test failure is now:

    invokeAutomatedTrigger WAS called for rightContext: add

@zixlin7 does this indicate a CW bug, or a test bug? 

Related: https://github.com/aws/aws-toolkit-vscode/commit/df9fc7315b37844a09b7f60b49c604fa267a88ab